ESP_SHT3x/ESP_SHT3x.ino

```diff
@@ -1,12 +1,79 @@
+#include <ESP8266WiFi.h>
+#include <ESP8266mDNS.h>
+#include <WiFiUdp.h>
+#include <ArduinoOTA.h>
 #include <Arduino.h>
 #include <Wire.h>
 #include "Adafruit_SHT31.h"
+
+const char* ssid = "TP-Link_4346";
+const char* password = "84850034";
```

Comment:
Please make a habit of never committing credentials. For now, remove them prior to merge.